certbot/tests/cli_test.py

```diff
@@ -406,19 +431,22 @@ class SetByCliTest(unittest.TestCase):
     def setUp(self):
         reload_module(cli)
 
-    def test_webroot_map(self):
+    @test_util.patch_get_utility()
```

Comment:
Similarly, it'd be nice if we didn't have to decorate every method and add an unused parameter. I'd probably fix this by modifying `_call_set_by_cli`.